gwcs/coordinate_frames.py

```diff
@@ -0,0 +1,402 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Defines coordinate frames and ties them to data axes.
+"""
+from __future__ import division, print_function
+
+import numpy as np
+from astropy import time
+from astropy import units as u
+from astropy import utils as astutil
+from astropy import coordinates as coo
+from astropy.coordinates import (BaseCoordinateFrame, FrameAttribute,
+                                 RepresentationMapping)
+
+from . import spectral_builtin_frames
+from .spectral_builtin_frames import *
+from .representation import *
+
+
+__all__ = ['DetectorFrame', 'CelestialFrame', 'SpectralFrame', 'TimeFrame',
+           'CompositeFrame', 'FocalPlaneFrame']
+
+
+STANDARD_REFERENCE_POSITION = ["GEOCENTER", "BARYCENTER", "HELIOCENTER",
+                               "TOPOCENTER", "LSR", "LSRK", "LSRD",
+                               "GALACTIC_CENTER", "MOON", "LOCAL_GROUP_CENTER"]
+
```

Comment:
Docstrings for module-level constants currently are not working.  I discovered that here in `astropy`: https://github.com/astropy/astropy/pull/3208.

It's now an open issue: https://github.com/astropy/astropy-helpers/issues/117.